api/src/main/java/org/pmiops/workbench/rdr/RdrMapper.java

```diff
@@ -97,8 +98,7 @@ default void addOtherPopulationDetails(DbWorkspace source, @MappingTarget RdrWor
   @Mapping(source = "dbUser.demographicSurvey.lgbtqIdentity", target = "lgbtqIdentity")
   @Mapping(source = "dbUser.demographicSurvey.identifiesAsLgbtq", target = "identifiesAsLgbtq")
   @Mapping(source = "accessTiers", target = "accessTierShortNames")
-  // re-enable with feature flag in RW-8355
-  // @Mapping(source = "dbUser.demographicSurveyV2", target = "demographicSurveyV2")
+  @Mapping(source = "dbUser.demographicSurveyV2", target = "demographicSurveyV2")
```

Comment:
Did you want a feature flag here? I am not sure, but I was looking add your old comment.

Reply:
I changed my approach: instead of applying the FF here, we remove the Demo Survey after the mapping: in RdrExportServiceImpl.toRdrResearcher()

Alternatively we could remove it before the mapping.